docs/localized/JP/Basic-Guide.md

```diff
@@ -0,0 +1,112 @@
+4. **Project** windowより`Assets/ML-Agents/Examples/3DBall/TFModels` folderの項目を開いてください。
+5. `TFModels` folderに入っている、`3DBall` model file をドラッグ&ドロップで**Ball3DBrain** object の **Brain** componentにある**Graph Model** fieldにセットをしてください・
+5. **Play** buttonをクリックすると、Gameが開始して各エージェントが先ほどセットをしたPretrained Modelをもとに行動をしていきます。
+
+![Running a pretrained model](images/running-a-pretrained-model.gif)
+
+## Basics Jupyter Notebookを使ってみましょう
+
+`python/Basics` [Jupyter notebook](Background-Jupyter.md)はPython APIの使い方を説明しているものとなり、こちらを一度ご覧いただくと今後の理解が捗ると思います。
+また、こちらに入っているコードを使い簡単な環境のテストを行うこともできます。その際には`env_name`を ゲームユニットの名前と合わせるようにしてください。
+ゲームユニットの作り方はこちらです。  
+[use an executable](Learning-Environment-Executable.md)   
+また、`env_name`を`None`にした場合は、Unityで開いているSceneの方が対象環境と認識されます。  
+
+追加情報に関してはこちらを参照ください・  
+[Python API](Python-API.md)
+
+## Reinforcement Learningを使用してBrainをトレーニングしましょう
+### BrainをExternalにセットしましょう
+これまで、我々はPretrained Modelを使用してきました。その際にはBrainのコマンドを決定する主体は先ほどのPretrained Modelなので、内部に内包されている形でした。
+ですので、Brain typeが Internal出会ったのです。
+しかし、ここからは、このページてみたように、`jupyter Notebook`を使用したり、後ほど紹介する`learn.py`などを用いて、Unityをは別に動いているプロセスである、pythonからのコマンドを受け取って、
+Agentsを動かして胃トレーニングを進めていきます。
+よって、Brain typeの方を **External**に切り替えてください。
+
+![Set Brain to External](images/mlagents-SetExternalBrain.png)
+
+### Training the environment
+これだけで準備は完了です。実際にPythonのスクリプトを動かして、Trainingを進めていきましょう。
+1. Terminal/CMDを開いてください。 
+2. ML-Agentsのgithub repo配下のpythonのフォルダーに移動してください。`./ml-agent/pythonpython`  
+4. Run `python3 learn.py --run-id=<run-identifier> --train`
+詳細:
+- `<run-identifier>` : トレーニングの名称。この名前がトレーニング終了後にmodelやそのトレーニングの詳細を含むフォルダーを形成する。
+- `--train` : このフラグにより、トレーニングであることを明示する。
+5. _"Start training by pressing the Play button in the Unity Editor"_ が表示されたら、Unity上からGameを起動してください。
+これでトレーニングが始まります。
+
+**Note**: また、前述のExecutableという、buildされた単体のGameからも同様の Interfaceでトレーニングを行なっていくことができます。詳細は下記になります。  
+ [this page](Learning-Environment-Executable.md)
+
+![Training command example](images/training-command-example.png)
+
+
+`learn.py`が正常に起動をすると、下記の画面が見えるはずです。
+
+![Training running](images/training-running.png)
+
+### After training
+トレーニングについては、いつでもストップをすることができます。ctrl+cか、UnityのGameを直接止めることができます。  
+トレーニング終了後、`ml-agents/python/models/<run-identifier>/editor_<academy_name>_<run-identifier>.bytes` というファイルが生成されます。  
+`<academy_name>` については、皆様のSceneで使われている名前が採用されます。  
+これでついに、さきほどおこなったPretrained Modelと同じ手順で、今回生成したモデルを使用して、Agentsを動かしていくことができます。  
+[Pretrained Modelの使い方詳細](#play-an-example-environment-using-pretrained-model).  
+
+1. `ml-agents/python/models/<run-identifier>/editor_<academy_name>_<run-identifier>.bytes`を`unity-environment/Assets/ML-Agents/Examples/3DBall/TFModels/`配下に格納してください。
+2. Unityから**3DBall** sceneを選んでください。
+3. **Ball3DBrain** object を選びます。
+4. **Type of Brain** を **Internal**に変更しましょう。
+5. `<env_name>_<run-identifier>.bytes` file をドラッグ&ドロップで **Brain** にセットしましょう。
+6. Playボタンを押してみましょう。
+これで、先ほど作成したmodelを実際のAgentsに適用をすることができました。
+
+## Next Steps
+
```

Comment:
@Rowing0914 Hi, we appreciate your effort to translate this document!
We'd like to update your translation as:
1. finish partially translated file to be completely translated files.
2. for files you've already finished translation, add some comments to get aligned.
3. leave wholly untranslated ones as untranslated (in English)

Here I'd like to start with this file, in rule 1.